strawberryfields/ops.py

```diff
@@ -73,11 +73,11 @@
 * :class:`Measurement` operations manipulate the register state and produce classical information.
   The information is directly available only after the simulation has been run up to the point of measurement::
 
-    eng, q = sf.Engine(2)
+    eng, (alice, bob) = sf.Engine(2)
     with eng:
-      Measure       | q[0]
+      Measure       | alice
       eng.run()
-      Dgate(q[0].val) | q[1]
+      Dgate(alice.val) | bob
```

Comment:
Was this something you changed before the program/engine split 😆

Reply:
Yeah. The rest of the module docstring still needs to be updated.